src/icons/languageIcons.ts

```diff
@@ -138,4 +138,5 @@ export const languageIcons: LanguageIcon[] = [
   { icon: { name: 'dart' }, ids: ['dart'] },
   { icon: { name: 'cadence' }, ids: ['cadence'] },
   { icon: { name: 'stylable' }, ids: ['stylable'] },
+  { icon: { name: 'jsonnet' }, ids: ['jsonnet'] },
```

Comment:
do we need an entry to `libsonnet` as well?

Reply:
> do we need an entry to `libsonnet` as well?

It is possible to set only one icon in the vscode language icon. And since .libsonnet is just a file extension and the language used is actually jsonnet, I set the main icon to jsonnet. It might be possible to somehow set 2 icons for the language, but right now, at least, I don't know.